WordPress/Classes/ViewRelated/Aztec/New Group/AztecVerificationPromptHelper.swift

```diff
@@ -0,0 +1,101 @@
+import UIKit
+
+internal class AztecVerificationPromptHelper: NSObject {
```

Comment:
Plus: If this class won't ever be used with anything in ObjC, you may drop the `NSObject` superclass, as well

Reply:
Unfortunately, it's gotta be `NSObject`, to be able to conform to `UIViewControllerTransitioningDelegate`:(

Reply:
Ahhhhhh gotcha!